org.eclipse.tm4e.registry/schema/grammars.exsd

```diff
@@ -1,112 +1,118 @@
-            <annotation>
-               <documentation>
-                  the id of a content type as defined by the &lt;code&gt;org.eclipse.core.runtime.contentTypes&lt;/code&gt; extension point for which this TextMate scope name should be used.
-               </documentation>
-            </annotation>
-         </attribute>
-      </complexType>
-   </element>
-
-   <annotation>
-      <appinfo>
-         <meta.section type="since"/>
-      </appinfo>
-      <documentation>
-         2.0
-      </documentation>
-   </annotation>
-
-
-
-   <annotation>
-      <appinfo>
-         <meta.section type="implementation"/>
-      </appinfo>
-      <documentation>
-         This plugin itself does not have any predefined builders.
-      </documentation>
-   </annotation>
-
-
-</schema>
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="org.eclipse.tm4e.core" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="org.eclipse.tm4e.core" id="grammars" name="TextMate grammars"/>
+      </appinfo>
+      <documentation>
+         Extension point to register TextMate grammars *.tmLanguage, *.json (VSCode format).
+      </documentation>
+   </annotation>
+
+   <element name="extension">
+      <annotation>
+         <appinfo>
+            <meta.element />
+         </appinfo>
+      </annotation>
+      <complexType>
+         <sequence minOccurs="1" maxOccurs="unbounded">
+            <element ref="grammar" minOccurs="0" maxOccurs="unbounded"/>
+            <element ref="scopeNameContentTypeBinding" minOccurs="0" maxOccurs="unbounded"/>
+         </sequence>
+         <attribute name="point" type="string" use="required">
+            <annotation>
+               <documentation>
+                  a fully-qualified name of the target extension point
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="id" type="string">
+            <annotation>
+               <documentation>
+                  an optional id
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="name" type="string">
+            <annotation>
+               <documentation>
+                  an optional name
+               </documentation>
+            </annotation>
+         </attribute>
+      </complexType>
+   </element>
+
+   <element name="grammar">
+      <annotation>
+         <documentation>
+            This extension point allows developers to register TextMate grammars.
+         </documentation>
+      </annotation>
+      <complexType>
+         <attribute name="scopeName" type="string" use="required">
+            <annotation>
+               <documentation>
+                  TextMate scope name.
+               </documentation>
+            </annotation>
+         </attribute>
+         <attribute name="path" type="string">
```

Comment:
It's not your error. It's my fault. "path" attribute has wrong label. I will fix it.